site_scons/prereq_tools/base.py

```diff
@@ -600,16 +600,22 @@ def __init__(self, env, variables, config_file=None, arch=None):
         self.__opts = variables
         self.configs = None
 
-        real_env = self.__env['ENV']
-
-        for var in ["HOME", "TERM", "SSH_AUTH_SOCK",
-                    "http_proxy", "https_proxy",
-                    "PKG_CONFIG_PATH", "MODULEPATH",
-                    "MODULESHOME", "MODULESLOADED",
-                    "I_MPI_ROOT"]:
-            value = os.environ.get(var)
-            if value:
-                real_env[var] = value
+        self.add_opts(EnumVariable('SCONS_ENV', "Default SCons environment inheritance",
+                                   'minimal', ['minimal', 'full'], ignorecase=1))
+
+        if self.__env.get("SCONS_ENV") == 'full':
```

Comment:
I wonder if we should move the VIRTUAL_ENV code here from https://github.com/daos-stack/daos/blob/master/SConstruct#L349-L352 then you could have 'minimal', 'venv', 'full' with the default being venv.  We also have some special handling of PATH for mpicc elsewhere in the tree